expense-rescript/src/context/GlobalContext.res

```diff
@@ -0,0 +1,59 @@
+@react.component
+type transaction = {id: int, text: string, price: int}
```

Comment:
Why do we have `id` ? Are we using the `id` except for passing as keys?

Reply:
Make a type file name `Transaction`
```
type t = {
   id: int,
   text: string,
   amount: string
}
```

Reply:
> Why do we have `id` ? Are we using the `id` except for passing as keys?

No